packages/client/README.md

```diff
@@ -10,97 +10,188 @@
 
 [![Build status](https://github.com/streamr-dev/monorepo/actions/workflows/client-build.yml/badge.svg)](https://github.com/streamr-dev/monorepo/actions/workflows/client-build.yml)
 ![latest npm package version](https://img.shields.io/npm/v/streamr-client?label=latest)
-[![GitHub stars](https://img.shields.io/github/stars/streamr-dev/network-monorepo?style=social)
+![GitHub stars](https://img.shields.io/github/stars/streamr-dev/network-monorepo?style=social)
 [![Discord Chat](https://img.shields.io/discord/801574432350928907.svg?label=Discord&logo=Discord&colorB=7289da)](https://discord.gg/FVtAph9cvz)
 
 This library allows you to easily interact with the [Streamr Network](https://streamr.network) from JavaScript-based environments, such as browsers and [node.js](https://nodejs.org). The library wraps a Streamr light node for publishing and subscribing to data, as well as contains convenience functions for creating and managing streams.
 
 Please see the [Streamr project docs](https://streamr.network/docs) for more detailed documentation.
 
+## Contents
```

Comment:
This could contain links to the relevant sections. Also we could include e.g. two levels of indentation to keep this simple.

Reply:
Links were added but reducing to max two level indentation not yet? And Setup subsections can be left out from this TOC.